crates/common/consensus/src/execution_engine/blob_versioned_hashes.rs

```diff
@@ -12,10 +12,10 @@ pub fn blob_versioned_hashes(execution_payload: &ExecutionPayload) -> anyhow::Re
             .map_err(|err| anyhow!("Failed to detect transaction type: {err:?}"))?
             == TransactionType::BlobTransaction
         {
-            let blob_transaction = BlobTransaction::decode(&mut &transaction[1..])?;
-            blob_versioned_hashes.extend(blob_transaction.blob_versioned_hashes);
+            if let Ok(blob_transaction) = BlobTransaction::decode(&mut &transaction[1..]) {
```

Comment:
Shouldn't this throw an error? If it purports to be a blob transaction but fails to decode, it should return an error imo.

Also, can we pass a list of transactions of `TransactionType::BlobTransaction` to `is_data_available`? `is_data_available` doesn't need to know all of execution payload.